packages/components/src/Avatar/src/AnonymousAvatar.tsx

```diff
@@ -0,0 +1,76 @@
+import { AnonymousRichIcon } from "@hopper-ui/icons";
+import { type ResponsiveProp, useStyledSystem, type StyledSystemProps, useResponsiveValue } from "@hopper-ui/styled-system";
+import clsx from "clsx";
+import { type CSSProperties, forwardRef, type ForwardedRef } from "react";
+import { useContextProps } from "react-aria-components";
+
+import type { BaseComponentProps } from "../../utils/index.ts";
+
+import type { AvatarSize } from "./Avatar.tsx";
+import { AvatarContext } from "./AvatarContext.ts";
+import { RichIconAvatarImage } from "./RichIconAvatarImage.tsx";
+
+export const GlobalAnonymousAvatarCssSelector = "hop-AnonymousAvatar";
+
+export interface AnonymousAvatarProps extends StyledSystemProps, BaseComponentProps {
+    /**
+     * The label to display for the avatar.
+     */
+    "aria-label": string;
```

Comment:
Adding this property as required is an orbiter thing. in RAC, they prefer to keep it optional, but add a warning like this inside the component: 

```

  if (!props['aria-label'] && !props['aria-labelledby']) {
    console.warn('An aria-label or aria-labelledby prop is required for accessibility.');
  }
```